mysqld_exporter.go

```diff
@@ -170,6 +173,76 @@ func init() {
 	prometheus.MustRegister(version.NewCollector("mysqld_exporter"))
 }
 
+func filter(filters map[string]bool, name string) bool {
+	flg := flag.Lookup("collect." + name)
```

Comment:
Minor nit, I think the golang style for this would just be `f`.

Reply:
I had used `f` in the original implementation, but changed it to `flg` later 😄 
I'll change it back.